src/Yoti/ActivityDetails.php

```diff
@@ -4,10 +4,13 @@
 use Yoti\Entity\Selfie;
 use Yoti\Entity\Profile;
 use Yoti\Entity\Attribute;
+use Attrpubapi_v1\Attribute as ProtobufAttribute;
 use Attrpubapi_v1\AttributeList;
 use Yoti\Helper\ActivityDetailsHelper;
 use Yoti\Util\Age\AgeUnderOverProcessor;
 use Yoti\Util\Profile\AnchorProcessor;
+use Yoti\Util\Profile\AttributeConverter;
+use Yoti\Exception\AttributeException;
```

Comment:
Appears to be unused

Reply:
Yes I will remove that, I din't want to restrict the catch on `getYotiAttribute` to this exception only and forgot to remove it.